chrome/app.js

```diff
@@ -31,11 +31,11 @@ GmailToTrello.App.prototype.bindEvents = function() {
         gtt_log("Status: " + Trello.authorized().toString());
     });
     
-    this.model.event.addListener('onBeforeLoadTrello', function() {
+    this.model.event.addListener('onBeforeLoadPopup', function() {
```

Comment:
Renamed LoadTrello to LoadPopup since we're now requesting Trello data on page load, not just on popup load.